targetplatform/JBTIS Oxygen Linux x86_64.launch

```diff
@@ -15,14 +15,14 @@
 <booleanAttribute key="includeOptional" value="true"/>
 <stringAttribute key="location" value="./ws_fusetooling_dev"/>
 <booleanAttribute key="org.eclipse.jdt.launching.ATTR_USE_START_ON_FIRST_THREAD" value="true"/>
-<stringAttribute key="org.eclipse.jdt.launching.JRE_CONTAINER" value="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/java-8-openjdk-amd64"/>
+<stringAttribute key="org.eclipse.jdt.launching.JRE_CONTAINER" value="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/java-1.8.0-openjdk-1.8.0.141-1.b16.fc26.x86_64"/>
```

Comment:
because I renamed it to fit how u named it on Windows and to reflect we are no longer using JBTIS TP